features/DocumentController.feature

```diff
@@ -346,6 +346,25 @@ Feature: Document Controller
     And The document "document-1" content match:
       | name | "replaced1" |
 
+    @mappings
+  Scenario: CreateOrReplace multiple documents and return response without _source
+    Given an existing collection "nyc-open-data":"yellow-taxi"
+    And I "create" the following documents:
+      | _id          | body                               |
+      | "document-1" | { "name": "document1", "age": 42 } |
+    When I successfully execute the action "document":"mCreateOrReplace" with args:
+      | index      | "nyc-open-data"        |
+      | collection | "yellow-taxi"          |
+      | body       | { "documents": [ { "_id": "document-1", "body": { "name": "replaced1" } } ] } |
+      | source     | false                  |
+    Then I should receive a "successes" array of objects matching:
+      | _id          |  status | result    | created |
```

Comment:
You should test that `_source` is `undefined`. You can use the `_UNDEFINED_` placeholder for this